frontend/public/components/nav/nav-header.tsx

```diff
@@ -41,7 +41,9 @@ const PerspectiveDropdownItem: React.FC<PerspectiveDropdownItemProps> = ({
     >
       <Title headingLevel="h2" size="md" data-test-id="perspective-switcher-menu-option">
         <span className="oc-nav-header__icon">
-          <LazyIcon />
+          <React.Suspense fallback={<span>&emsp;</span>}>
```

Comment:
Another tiny nit: sorry : p
Fragment might be better here:
```suggestion
          <React.Suspense fallback={<>&emsp;</>}>
```

Reply:
ty, changed it to fragment :)